app/views/registers/show.html.haml

```diff
@@ -44,9 +44,20 @@
             = formatted_date(Array(@register_data.get_entries).last.timestamp)
             %strong.text-spacing -
             = link_to "See all register updates", history_path(@register.slug)
+          - if @register.related_registers.present?
+            %dt Related registers:
+            %dd
+              != @register.related_registers
```

Comment:
Are you wanting to output unescaped HTML here?

Reply:
`!=` is the same as `.html_safe`